doc/modules/outlier_detection.rst

```diff
@@ -112,14 +107,33 @@ an ellipse. For more details on the different estimators refer to the example
 :ref:`sphx_glr_auto_examples_plot_anomaly_comparison.py` and the sections
 hereunder.
 
+.. figure:: ../auto_examples/images/sphx_glr_plot_anomaly_comparison_001.png
+   :target: ../auto_examples/plot_anomaly_comparison.html
+   :align: center
+   :scale: 50
```

Comment:
why did you change the location of this figure? IMO it should stay at the same position.